slam-ws/src/nav_goal/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package format="2">
+
+  <name>nav_goal</name>
+  <version>0.0.0</version>
+  <description>The nav_goal package allows publishing to the move_base/goal topic to tell the robot where to move </description>
+  <maintainer email="federico.pizarrobejarano@mail.utoronto.ca">federico</maintainer>
+  <license>TODO</license>
```

Comment:
what does TODO mean ?

Reply:
The license tag is required, so we can't remove it or else the build will fail. It's there so you can specify what kind of license your software has (BSD, MIT, etc) so that other people who want to use your software know if its open, restricted, completely copyright, etc. If we wanted to specify a license we would probably use BSD. You can see more here: http://wiki.ros.org/ROS/Tutorials/catkin/CreatingPackage#license_tags and here: https://opensource.org/licenses/alphabetical. If we want to use a license I propose the most open one: https://opensource.org/licenses/0BSD. However doubt it is necessary because no one is going to use our software

Reply:
Up to y'all if we want to specify @AbiJaya1998 @erickmu1

Reply:
Ok just wanted to know so we can decide on it later .... nothing urgent

Reply:
lets leave it the same for now

Reply:
Yeah sounds good. I just checked UTRA-ART and it seems that sometimes people put in licenses, sometimes they don't. The licenses they've used are MIT, BSD, and Apache 2.0 so it seems that its not really standardized throughout the club. @chandra-gummaluru @1999michael any idea on licensing? Doubt it matters but idk if there are any guidelines from the club.